target_chains/cosmwasm/contracts/pyth/src/contract.rs

```diff
@@ -469,7 +468,7 @@ fn update_price_feed_if_new(
                     arrival_time: env.block.time,
                     arrival_block: env.block.height,
                     price_feed,
-                    attestation_time,
+                    publish_time,
```

Comment:
I think this should remain the attestation_time, we are storing publish time inside the pricefeed right?

Reply:
storing price feed instead of price info. no need of this anymore